testsuite/integration/microprofile-tck/pom.xml

```diff
@@ -200,6 +199,15 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <excludes>
+                        <exclude>**/metrics/**</exclude>
+                    </excludes>
+                </configuration>
+            </plugin>
```

Comment:
What's this about? I don't think we should be excluding tests from the other MP TCKs.

Reply:
@bstansberry, the Fault Tolerance TCK has tests that exercise the _optional_ use of MP Metrics with FT. Without this exclusion, those tests run, and the TCKs fail. I think the exclusion is needed, but I can move it into the FT module if you'd prefer.